http-client/src/lib.rs

```diff
@@ -1,3 +1,15 @@
+#[cfg(all(feature = "tokio1", feature = "tokio02"))]
+compile_error!("feature \"tokio1\" and \"tokio02\" are mutably exclusive");
```

Comment:
```suggestion
compile_error!("""feature "tokio1" and "tokio02" are mutably exclusive""");
```

Reply:
it doesn't work properly the error message becomes:

`error: compile_error! takes 1 argument
 --> http-client/src/lib.rs:5:1
  |
5 | compile_error!("""feature "tokio1" or "tokio02" must be enabled for this crate""");
`

Reply:
hahah, ok, sorry for the noise then. I did run `cargo check` but I guess that's not enough.

Reply:
right, you have to do `cargo check  -p jsonrpsee-http-client --no-default-features` or `cargo check  -p jsonrpsee-http-client --features tokio02`

Because `tokio1` is enabled by default, feature flags life you know.

Reply:
We could do compile_error!("feature `tokio1` or `tokio02` must be enabled for this crate"); instead if you don't like the escaping (`\"`)